cygnus-common/src/test/java/com/telefonica/iot/cygnus/backends/http/HttpBackendTest.java

```diff
@@ -137,5 +146,40 @@ public void testDoRequestWithArrayResponse() throws Exception {
             throw e;
         } // try catch
     } // testDoRequestWithArrayResponse
+    
+    /**
+     * [HttpBackend.createJsonResponse] -------- A JsonResponse object is not crated if the content-type header does
+     * not contains 'application/json'.
+     */
+    @Test
+    public void testCreateJsonResponseNotJsonPayload() {
+        System.out.println(getTestTraceHead("[HttpBackend.createJsonResponse]")
+                + "-------- A JsonResponse object is not crated if the content-type header does not contains "
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1245/commits/405cb10f589524379191ce71fe799695a601450d